hyperspy/_signals/signal2d.py

```diff
@@ -650,6 +649,74 @@ def add_ramp(self, ramp_x, ramp_y, offset=0):
         ramp += ramp_y * yy
         self.data += ramp
 
+    def find_peaks2D(self, method='skimage', *args, **kwargs):
```

Comment:
This needs the usual `show_progressbar` and `parallel` kwargs that come with `map` methods.

Also, tests should probably parametrized for both `parallel` and `lazy`